admin/RELEASE.md

```diff
@@ -1,31 +1,30 @@
 # Release Process
 
 > Documentation guide based on the releases of `4.0.5` and `4.1.0` on January 31, 2021.
-> Updated for `4.1.2` on May 17, 2021.
+>
+> Updated for `4.1.6` on December 24, 2021.
+>
 > -MGatner
 
-## Labeling PRs
+## Changelog
 
-To auto-generate changelog, each PR to be listed in changelog must have one of the following [labels](https://github.com/codeigniter4/CodeIgniter4/labels):
+When generating the changelog each Pull Request to be included must have one of the following [labels](https://github.com/codeigniter4/CodeIgniter4/labels):
 - **bug** ... PRs that fix bugs
 - **enhancement** ... PRs to improve existing functionalities
 - **new feature** ... PRs for new features
 - **refactor** ... PRs to refactor
 
-And PRs that have the breaking changes must have the following label:
+PRs with breaking changes must have the following additional label:
 - **breaking change** ... PRs that may break existing functionalities
 
+To auto-generate, start drafting a new Release and use the "Auto-generate release notes" button.
```

Comment:
How do we do? It seems it needs a valid tag. When do we tag?
![Screenshot 2021-12-25 9 19 19](https://user-images.githubusercontent.com/87955/147374770-d213f06d-cea3-4c9d-8e3c-cf62bb79e134.png)

Reply:
At this stage you won't actually complete the release, so you can type anything into the tag dropdown.